File: exercise/sentinel.py
userinput = int(input("What is your value: "))
sumed = 0

while userinput > 0:

    sumed+=userinput
    userinput = int(input("Input another value or enter 0 to see sum of all values enterd: "))    

print("\nYour sumed value is: ", sumed, "\n")

File: exercise/sentinel.c
#include <stdio.h>

int sumed;
int input;
int main(){

    printf("What is your value: ");
    scanf("%d", &input);

    while(input > 0){
       
        sumed+=input;
        printf("What is your value: ");
        scanf("%d", &input);

    }

    printf("\nYour sumed value is ==> %d",sumed);

}
